how to log to aws inside container (#810)
ferrocene/ferrocene: ferrocene/doc/internal-procedures/src/working-with-the-ci.rst

For Mac or Windows jobs, you can coarsely reproduce the CI by running the
lines in ``SCRIPT``, line by line if desired.

For Linux jobs, before running the container, make sure you are logged 
in to ``aws`` with ``aws sso login --profile ferrocene-ci``.
Then enter the Docker container specified by the ``executor`` line. Mount the 

> ferrocene directory as well as your identifier for ``aws``. 

Reviewer: ```suggestion
ferrocene directory as well as the directory used by ``aws`` tool. 
```

.. code-block:: bash

   docker run --rm --tty --interactive --workdir /ferrocene \
      --mount "type=bind,src=$(pwd),dst=/ferrocene" \
      --mount "type=bind,src=$(HOME/.aws/),dst=/home/ci/.aws" \
      ubuntu-20 bash

Inside the container, run:

.. code-block:: bash

>    aws sts get-caller-identity --profile ferrocene-ci

Reviewer 2: This is wrong, it doesn't do anything relevant here.
